plugins/jira/commands/status-rollup.md

```diff
@@ -1,6 +1,7 @@
 ---
 description: Generate a weekly status rollup comment for any JIRA issue based on all child issues
 argument-hint: issue-id [--start-date YYYY-MM-DD] [--end-date YYYY-MM-DD]
+version: 1.0
```

Comment:
instead of duplicating a source for version, can we relying on the existing plugin metadata?